Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -258,6 +260,25 @@ - (void)removeChatWithAllMessages:(OCTChat *)chat
     });
 }
 
+- (void)removeAllCalls
+{
+    RLMResults *calls = [OCTCall objectsInRealm:self.realm where:nil];
+
+    DDLogInfo(@"OCTRealmManager: removing %lu calls", calls.count);
+
+    RBQRealmChangeLogger *logger = [self logger];
+
+    for (OCTCall *call in calls) {
+        OCTMessageCallEvent event = (call.status != OCTCallStatusActive) ? OCTMessageCallEventUnanswered : OCTMessageCallEventAnswered;
```

Comment:
It would look nicer if it would be positive statement

```
OCTMessageCallEvent event = (call.status == OCTCallStatusActive) ? OCTMessageCallEventAnswered : OCTMessageCallEventUnanswered; 
```